rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -807,14 +832,23 @@ public ResponseEntity patchReleases(
                             // TODO: Add example for MAP value
                     }
             )
-            @RequestBody Object releaseURIs
-    ) throws URISyntaxException, TException {
-        RequestStatus patchReleasesStatus = addOrPatchReleasesToProject(id, releaseURIs, true);
-        if (patchReleasesStatus == RequestStatus.SENT_TO_MODERATOR) {
-            return new ResponseEntity<>(RESPONSE_BODY_FOR_MODERATION_REQUEST, HttpStatus.ACCEPTED);
-        }
-        return new ResponseEntity<>(HttpStatus.CREATED);
-    }
+            @RequestBody Object releaseURIs,
+            @Parameter(description = "comment message.")
+            @RequestParam(value = "comment", required = false) String comment
+	) throws URISyntaxException, TException {
+		final User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+		Project project = projectService.getProjectForUserById(id, sw360User);
+		if (!restControllerHelper.isWriteActionAllowed(project, sw360User) && comment == null) {
+			return new ResponseEntity(RESPONSE_BODY_FOR_MODERATION_REQUEST_WITH_COMMIT, HttpStatus.ACCEPTED);
```

Comment:
In case of missing request parameter, shouldn't the response status be `BAD_REQUEST`?